injector/ProgramProcessor.py

```
@@ -35,11 +34,38 @@ def __init__(self, sourceFile, workingDirectory, sysinfo):
+
+    def mapInjectedSourceLineno(self, tree):
+        '''
+            This function maps the lineno from the original
+            source to the injected source.
+        '''
+        # Save the position of the original code 
+        count = 0
+        mapped = {}
+        for node in ast.walk(tree):
+            if hasattr(node, "logTypeCount") and hasattr(node, "lineno"):
+                mapped[str(count)] = getattr(node, "logTypeCount")
```

Ruff (0.11.9)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)